truss/patch/calc_patch.py

```diff
@@ -16,6 +18,7 @@
 from truss.truss_config import TrussConfig
 from truss.truss_spec import TrussSpec
 
+logger: logging.Logger = logging.getLogger(__name__)
 PYCACHE_IGNORE_PATTERNS = [
```

Comment:
should we add .DS_STORE to these patterns? 
It might also be a good idea to grab a more comprehensive list like this one? https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
I'm not sure if it's worth it at this point. If the file is created inside the model directory then we can simply patch it, if at the top level we ignore it. The only remaining cases were if they are created under data or packages directories, which doesn't seem like a common case.